<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1040</width>
    <height>690</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>My notes</string>
  </property>
  <widget class="QLabel" name="label_5">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>20</y>
     <width>1031</width>
     <height>721</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap>../../Desktop/other/bg.png</pixmap>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>950</x>
     <y>-40</y>
     <width>91</width>
     <height>771</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap>../../Desktop/other/right_border4.png</pixmap>
   </property>
  </widget>
  <widget class="QLabel" name="label_7">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>211</width>
     <height>91</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap>../../Desktop/other/smth2.png</pixmap>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>0</y>
     <width>731</width>
     <height>91</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap>../../Desktop/other/top_border2png.png</pixmap>
   </property>
  </widget>
  <widget class="QLabel" name="label_6">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>221</width>
     <height>691</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap>../../Desktop/other/left_border.png</pixmap>
   </property>
  </widget>
  <widget class="QPushButton" name="settings_btn">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>30</y>
     <width>61</width>
     <height>61</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="icon">
    <iconset>
     <normaloff>../../Desktop/other/settings5.png</normaloff>../../Desktop/other/settings5.png</iconset>
   </property>
   <property name="iconSize">
    <size>
     <width>50</width>
     <height>60</height>
    </size>
   </property>
  </widget>
  <widget class="QPushButton" name="add_group_btn">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>30</y>
     <width>61</width>
     <height>61</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="icon">
    <iconset>
     <normaloff>../../Desktop/other/group2.png</normaloff>../../Desktop/other/group2.png</iconset>
   </property>
   <property name="iconSize">
    <size>
     <width>51</width>
     <height>51</height>
    </size>
   </property>
  </widget>
  <widget class="QListWidget" name="group_list">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>110</y>
     <width>201</width>
     <height>571</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="delete_group_btn">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>30</y>
     <width>61</width>
     <height>61</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="icon">
    <iconset>
     <normaloff>../../Desktop/other/del group.png</normaloff>../../Desktop/other/del group.png</iconset>
   </property>
   <property name="iconSize">
    <size>
     <width>51</width>
     <height>51</height>
    </size>
   </property>
  </widget>
  <widget class="QListWidget" name="note_list">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>120</y>
     <width>711</width>
     <height>561</height>
    </rect>
   </property>
   <property name="wordWrap">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>80</y>
     <width>731</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap>../../Desktop/other/horizontal_line2.png</pixmap>
   </property>
  </widget>
  <widget class="QPushButton" name="add_note_btn">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>80</y>
     <width>81</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>New</string>
   </property>
   <property name="icon">
    <iconset>
     <normaloff>../../Desktop/other/plus3.png</normaloff>../../Desktop/other/plus3.png</iconset>
   </property>
   <property name="iconSize">
    <size>
     <width>20</width>
     <height>20</height>
    </size>
   </property>
  </widget>
  <widget class="QPushButton" name="edit_note_btn">
   <property name="geometry">
    <rect>
     <x>300</x>
     <y>80</y>
     <width>81</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Edit</string>
   </property>
   <property name="icon">
    <iconset>
     <normaloff>../../Desktop/other/pen.png</normaloff>../../Desktop/other/pen.png</iconset>
   </property>
   <property name="iconSize">
    <size>
     <width>22</width>
     <height>22</height>
    </size>
   </property>
  </widget>
  <widget class="QPushButton" name="delete_note_btn">
   <property name="geometry">
    <rect>
     <x>380</x>
     <y>80</y>
     <width>81</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Delete</string>
   </property>
   <property name="icon">
    <iconset>
     <normaloff>../../Desktop/other/trash3.png</normaloff>../../Desktop/other/trash3.png</iconset>
   </property>
   <property name="iconSize">
    <size>
     <width>23</width>
     <height>23</height>
    </size>
   </property>
  </widget>
  <widget class="QPushButton" name="done_btn">
   <property name="geometry">
    <rect>
     <x>870</x>
     <y>80</y>
     <width>81</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>done</string>
   </property>
   <property name="icon">
    <iconset>
     <normaloff>../../Desktop/other/check1.png</normaloff>../../Desktop/other/check1.png</iconset>
   </property>
   <property name="iconSize">
    <size>
     <width>25</width>
     <height>30</height>
    </size>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
